backend/spring-boot/src/main/java/org/bootstrapbugz/backend/auth/jwt/service/impl/AccessTokenServiceImpl.java

```diff
@@ -60,13 +62,15 @@ private void verifyToken(String token) {
     try {
       JwtUtil.verify(token, secret, PURPOSE);
     } catch (RuntimeException e) {
-      throw new UnauthorizedException(e.getMessage());
+      final var code = messageService.getMessage("auth.tokenInvalid");
```

Comment:
This catch logic seems to be repeating a lot, maybe you can move it to CustomExceptionHandler? 
Bonus effect is that you get to remove all the MessageService dependencies from the service classes